### ISSUE TITLE
Depracation warning in ember 2.1.0 beta

### ISSUE THREAD

**Issue by szimek** [ember-cli/ember-export-application-global#18]
I've just updated my app to ember 2.1.0 beta 3 and I'm getting the following warning:
```
DEPRECATION: The `initialize` method for Application initializer 'export-application-global' should take only one argument - `App`, an instance of an `Application`. 
[deprecation id: ember-application.app-initializer-initialize-arguments]
```
with the stack trace pointing to somewhere in the bundled vendor.js file. It looks like `initialize` method gets arguments [directly from `arguments` object](https://github.com/ember-cli/ember-export-application-global/blob/master/app/initializers/export-application-global.js#L5), probably for backwards compatibility.

**Comment by Kuzirashi** [comment]
This has been fixed in 1.0.4 version in [this commit](https://github.com/ember-cli/ember-export-application-global/commit/f273c6e1da727ca71b4111bcd723c0b7a6e0558a). Make sure you're using latest version. I've upgraded to 1.0.4 and deprecation is gone.

**Comment by rwjblue** [comment]
Thanks for confirming!
